website/source/guides/integrations/consul-connect/index.html.md

```diff
@@ -263,13 +254,12 @@ dashes (`-`) are converted to underscores (`_`) in environment variables so
 
 ## Limitations
 
-Prior to Nomad 0.10.0's final release, the Consul Connect integration has
-several limitations that have yet to be addressed:
-
- - Jobs with a `connect` stanza may not update properly. Workaround this by
-   stopping and starting Connect-enabled jobs.
+ - Consul Connect Native is not yet supported
+ - Consul Connect HTTP and gRPC checks are not yet supported
```

Comment:
@schmichael please review this section for correctness.

Reply:
Found a new one to add this morning:
```
 - Group services and checks do not yet support interpolation.
```